perf/benchmark/templates/fortio.yaml

```diff
@@ -83,9 +83,8 @@ spec:
         {{- if $.Values.includeOutboundIPRanges }}
         traffic.sidecar.istio.io/includeOutboundIPRanges: {{ $.Values.includeOutboundIPRanges }}
         {{- end }}
-        {{- if $.V.inject }}
         sidecar.istio.io/inject: "{{ $.V.inject }}"
-        {{- end }}
+        linkerd.io/inject: "{{ $.V.injectL }}"
```

Comment:
Did you mean to commit this here?

Reply:
Yes, make it possible to run with linkerd